petl/comparison.py

```diff
@@ -104,6 +104,26 @@ def _typestr(x):
 
 
 def comparable_itemgetter(*args):
-    f = operator.itemgetter(*args)
-    g = lambda x: Comparable(f(x))
+    getter = operator.itemgetter(*args)
+    getter_with_default = _itemgetter_with_default(*args)
+
+    def getter_with_fallback(x):
```

Comment:
- nitpicks:
  - parameter from `getter_with_fallback(x)` confuses with `lambda x`
  - `getter_with_fallback` is private? Should be `_getter_with_fallback`?

Reply:
Good point, pushed changes: https://github.com/petl-developers/petl/pull/601/commits/770812f8b6b7450ac0d350bcf457db05da8bb211

Reply:
LGTM